packages/node_modules/@webex/webex-core/test/unit/spec/services/services.js

```diff
@@ -210,6 +213,32 @@ describe('webex-core', () => {
           formattedHM.map((item) => item.name));
       });
     });
+
+    describe('#updateCredentialsConfig()', () => {
+      // updateCredentialsConfig must remove `/` if exist. so expected serviceList must be.
+      const expectedServiceList = {
+        idbroker: 'https://idbroker.webex.com',
+        identity: 'https://identity.webex.com'
+      };
+
+      beforeEach('get services list', async () => {
+        const servicesList = {
+          idbroker: 'https://idbroker.webex.com',
+          identity: 'https://identity.webex.com/'
+        };
+
+        catalog.list = sinon.stub().returns(servicesList);
+        await services.updateCredentialsConfig();
+      });
+
+      it('must update IDBROKER_BASE_URL as serviceList.idbroker', () => {
```

Comment:
To me, this reads as it is updating the environment variable. Perhaps a better naming structure for this test would be:
```suggestion
      it('sets the idbroker url properly when no trailing slash is present', () => {
```